### ISSUE TITLE
[Translation] 8_2_setting_up_runtime.md

### ISSUE THREAD

**Issue by Johnny4Fun** [cfsamson/book-exploring-async-basics#39]
> I find it easier to write about
this if we don't need to jump between functions too much and can just cover all this
logic from a-z.

This sentence is not easy for me to understand. Could you please clarify it?

**Comment by cfsamson** [comment]
It means that it's better for me to write one long function with all the logic chronologically since it's easier for me to explain in a book format, but I won't recommend writing such long functions in your own code since you should probably split it up in smaller functions with good descriptive names.

**Comment by Johnny4Fun** [comment]
How about the `cover all this logic from a-z`? It looks like a phrase.

**Comment by cfsamson** [comment]
Yeah, it's a phrase. It means "cover all this logic chronologically from the start to the end" ("a" is the start of the English alphabet, "z" is the end of the English alphabet)
